students/AY1617S2/lohJiaShunKenneth/lohJiaShunKenneth-Progress.md

```diff
@@ -6,7 +6,8 @@
 
 Effort| Type | PR | Issue
 :----:|:----:|:-----------|:------
-|||
++1 | L | - | Caption lab and Narration lab demo
++6 | L | - | Progress on SyncLab (Create sidebar, format list and fill copy)
```

Comment:
6 is not one of the acceptable numbers. 

Also, may be clean up your master branch. This PR has a lot of unrelated commits.